libs/cart/src/selectors/cart/cart.selector.ts

```diff
@@ -20,9 +20,132 @@ export interface DaffCartStateMemoizedSelectors<
 > {
 	selectCartState: MemoizedSelector<object, DaffCartReducerState<T>>;
 	selectCartValue: MemoizedSelector<object, T>;
-  selectCartLoading: MemoizedSelector<object, boolean>;
   selectCartResolved: MemoizedSelector<object, boolean>;
 
+  /**
+   * The object that holds all the loading states for cart operations.
+   */
+  selectCartLoadingObject: MemoizedSelector<object, DaffCartReducerState<T>['loading']>;
+  /**
+   * Selects whether there is a cart operation in progress.
+   * This does not include operations specifally for cart subfields.
+   */
+  selectCartLoading: MemoizedSelector<object, boolean>;
+  /**
+   * Selects whether there is a cart resolve operation in progress.
+   * This does not include operations specifally for cart subfields.
+   * This pertains only to requests that do not mutate data such as "load" or "list".
+   */
+  selectCartResolving: MemoizedSelector<object, boolean>;
+  /**
+   * Selects whether there is a cart mutate operation in progress.
+   * This does not include operations specifally for cart subfields.
+   * This pertains only to requests that mutate data such as "update".
+   */
+  selectCartMutating: MemoizedSelector<object, boolean>;
+  /**
+   * Selects whether there is a cart billing address operation in progress.
+   */
+  selectBillingAddressLoading: MemoizedSelector<object, boolean>;
+  /**
+   * Selects whether there is a cart billing address resolve operation in progress.
+   * This pertains only to requests that do not mutate data such as "load" or "list".
+   */
+  selectBillingAddressResolving: MemoizedSelector<object, boolean>;
+  /**
+   * Selects whether there is a cart billing address mutate operation in progress.
+   * This pertains only to requests that mutate data such as "update".
+   */
+  selectBillingAddressMutating: MemoizedSelector<object, boolean>;
```

Comment:
I think that these selectors should probably be named `selectCartBillingAddressMutating` instead of `selectBillingAddressMutating`, because of how they are sometimes imported into a class. Example:
```
const {
    selectBillingAddressMutating
} = getDaffCartSelectors();
```
This could create confusion and conflicts in the class

Reply:
The local var could always be aliased during destructuring to avoid conflicts:
```ts
const {
    selectBillingAddressMutating: selectCartBillingAddressMutating
} = getDaffCartSelectors();
```

As far as clarity, I think its probably fine either way. I was just following the pattern for the error selectors. If you think this should still be changed then the error selectors should also get the `Cart` prefix.

Reply:
No the aliasing sounds good to me